<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1622</width>
    <height>1047</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1622</width>
    <height>1047</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1622</width>
    <height>1047</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plugin configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
margin-top: 5px;
border: 2px solid gray;
border-color: #FF17365D;
border-radius: 9px;
}
QGroupBox::title {
    top: -8px;
    left: 10px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gb_menu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>88</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>88</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
margin-top: 5px;
border: 2px solid gray;
border-color: #FF17365D;
border-radius: 9px;
font-weight: bold;
}
QGroupBox::title {
    top: -8px;
    left: 10px;
    color: rgb(0, 250, 0);
}
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Load</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>61</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Load movie:</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_path_movie">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>241</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_load_movie">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>84</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Load movie</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Next/Previous</string>
        </property>
        <widget class="QPushButton" name="btn_next_step">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>10</y>
           <width>141</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Forward 1 step</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_prev_step">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>10</y>
           <width>141</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Backward 1 step</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Skip Frames</string>
        </property>
        <widget class="QTextEdit" name="tb_skip_frames">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>51</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>109</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Frames to skip:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_skip_backward">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>10</y>
           <width>111</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Backward steps</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_skip_forward">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>10</y>
           <width>101</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Forward steps</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="btn_close">
         <property name="geometry">
          <rect>
           <x>173</x>
           <y>20</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_export_coords">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Export Coords</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_all">
     <property name="minimumSize">
      <size>
       <width>1600</width>
       <height>935</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1600</width>
       <height>935</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="gb_img">
        <property name="minimumSize">
         <size>
          <width>1417</width>
          <height>904</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="lbl_img">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>15</y>
           <width>141</width>
           <height>66</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_coords">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 200, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPlainTextEdit" name="txt_coords">
         <property name="geometry">
          <rect>
           <x>7</x>
           <y>281</y>
           <width>136</width>
           <height>611</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_reset">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>117</y>
           <width>136</width>
           <height>37</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #ff0000;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Clear target</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_rect">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>15</y>
           <width>136</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>To TLBR</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_full">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>66</y>
           <width>136</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Full view target</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_apply_previous_draw">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>131</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
	color : #aaaaaa;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #56575a, stop: 1 #3a3b3e);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
         </property>
         <property name="text">
          <string>Apply previous draw</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
